tiledb/api/c_api/attribute/test/CMakeLists.txt

```diff
@@ -29,4 +29,5 @@ include(unit_test)
 commence(unit_test capi_attribute)
   this_target_sources(unit_capi_attribute.cc)
   this_target_object_libraries(capi_attribute_stub)
+  this_target_object_libraries(capi_attribute_stub)
```

Comment:
```suggestion
```

`capi_attribute_stub` needs to reference `capi_string` (same with the others).

Reply:
@teo-tsirpanis Could you fix since @kounelisagis is OOO?

Reply:
Fixed.